src/neuroglancer/annotation/annotation.ts

```diff
@@ -381,6 +363,7 @@ export abstract class MultiStepAnnotationTool extends PlaceAnnotationTool {
       }
     }
 
+    childCount = value.entries.length;
```

Comment:
is this unnecessary because of line 333?

Reply:
this should replace line 333, it was likely added after because the count can change between 333 - and 366